src/js/components/Grommet/__tests__/Grommet-test.js

```diff
@@ -21,3 +38,18 @@ test('Grommet hpe theme renders', () => {
   const tree = component.toJSON();
   expect(tree).toMatchSnapshot();
 });
+
+test('Grommet announces', (done) => {
+  const component = mount(
+    <Grommet><TestAnnouncer /></Grommet>
+  );
+  expect(component.getDOMNode()).toMatchSnapshot();
+
+  expect(document.body.querySelector('[aria-live]')).toMatchSnapshot();
+
+  setTimeout(() => {
+    // should clear the aria-live container
+    expect(document.body.querySelector('[aria-live]')).toMatchSnapshot();
+    done();
+  }, 600);
```

Comment:
Why 600? Maybe a comment, especially if this has to align with a timeout inside Grommet.